x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/pipelines_json_configurations.tsx

```diff
@@ -0,0 +1,142 @@
+  const { http } = useValues(HttpLogic);
+  const { pipelineNames, selectedPipeline, selectedPipelineId, selectedPipelineJSON } = useValues(
+    IndexPipelinesConfigurationsLogic
+  );
+  const { selectPipeline } = useActions(IndexPipelinesConfigurationsLogic);
+  return (
+    <>
+      <EuiSpacer />
+      <DataPanel
+        hasBorder
+        title={
+          <h2>
+            {i18n.translate(
+              'xpack.enterpriseSearch.content.indices.pipelines.tabs.jsonConfigurations.title',
+              { defaultMessage: 'Pipeline configurations' }
+            )}
+          </h2>
+        }
+        subtitle={i18n.translate(
+          'xpack.enterpriseSearch.content.indices.pipelines.tabs.jsonConfigurations.subtitle',
+          { defaultMessage: 'View the JSON for your pipeline configurations on this index.' }
+        )}
+        footerDocLink={
+          <EuiLink href={docLinks.ingestPipelines} target="_blank" color="subdued">
+            {i18n.translate(
+              'xpack.enterpriseSearch.content.indices.pipelines.tabs.jsonConfigurations.ingestionPipelines.docLink',
+              {
+                defaultMessage: 'Learn more about how Enterprise Search uses ingest pipelines',
+              }
+            )}
+          </EuiLink>
+        }
+        action={
+          pipelineNames.length > 0 && (
+            <EuiNotificationBadge size="m">{pipelineNames.length}</EuiNotificationBadge>
+          )
+        }
+        iconType="visVega"
+      >
+        <EuiFormRow
+          fullWidth
+          label={i18n.translate(
+            'xpack.enterpriseSearch.content.indices.pipelines.tabs.jsonConfigurations.selectLabel',
+            { defaultMessage: 'Select an ingest pipeline to view' }
+          )}
+        >
+          <EuiSelect
+            fullWidth
+            options={pipelineNames.map((name) => ({ text: name, value: name }))}
+            value={selectedPipelineId}
+            onChange={(e) => selectPipeline(e.target.value)}
+          />
+        </EuiFormRow>
+        <EuiSpacer size="m" />
+        {selectedPipeline && (
+          <>
+            <EuiFlexGroup alignItems="center">
+              <EuiFlexItem>
+                <PipelineJSONBadges />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                {isManagedPipeline(selectedPipeline) ? (
+                  <EuiButtonEmpty
+                    size="s"
+                    flush="both"
+                    iconType="eye"
+                    color="primary"
+                    href={http.basePath.prepend(
+                      `/app/management/ingest/ingest_pipelines/?pipeline=${selectedPipelineId}`
+                    )}
+                  >
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.content.indices.pipelines.tabs.jsonConfigurations.action.view',
+                      {
+                        defaultMessage: 'View in Stack Management',
```

Comment:
should we create a shared constant for this? we have it in the actions list for the ml inference pipelines as well?

Reply:
If it is for the translation, we are trying to keep them inline to make it consistent.

If it is for the action, I think that would be useful, similar to `EuiLinkTo`

Reply:
for the translation. I will leave it in place for consistency